doc/cs_registers.rst

```diff
@@ -282,6 +282,75 @@ Reset Value: ``0x0000_0000``
 | address[33:2]  |
 +----------------+
 
+.. _csr-dcsr:
+
+Debug Control and Status Register (dcsr)
+----------------------------------------
+
+CSR Address: ``0x7B0``
+
+Reset Value: ``0x4000_0003``
+
+Accessible in debug mode only.
+Ibex implements the following bit fields.
+Other bit fields read as zero.
+
++-------+-----+------------------------------------------------------------------+
+| Bit#  | R/W | Description                                                      |
++-------+-----+------------------------------------------------------------------+
+| 31:28 | R   | **xdebugver:** 4 = External spec-compliant debug support exists. |
++-------+-----+------------------------------------------------------------------+
+| 15    | RW  | **ebreakm:** EBREAK in M-Mode behaves as described in Privileged |
+|       |     | Spec (0), or enters Debug Mode (1).                              |
++-------+-----+------------------------------------------------------------------+
+| 12    | RW  | **ebreaku:** EBREAK in U-Mode behaves as described in Privileged |
```

Comment:
ebreaku was RW in the 0.13.2 spec, and changed to WARL in the latest debug spec (which is actually correct). Maybe describe it as WARL here and add a note that RW was an oversight in the 0.13.2 spec?

Reply:
Thanks for pointing this out @imphil . I adapted the description as suggested.